### DISCUSSION TITLE
Teslamate with a remotely-located car?

### DISCUSSION THREAD

**Discussion by rfgmacros** [teslamate-org/teslamate#3587] [Q&A] [answered]
We're going to loan our Model Y to a family member who lives in another state. I assume there's no (safe) way to have Teslamate keep tracking the car's charge, trips, etc.? It will *not* be possible to install and run Teslamate at the remote location; there's no way my relatives would be able to manage it, and I'm too remote to easily do so.

Note: This isn't about surveillance; they would be fully aware that the software was running. It's more about keeping an eye on the battery, their charging habits, etc. (This will be their first electric car.)

Is there some way that my locally-hosted Teslamate can pull the data from the Y located in another state?

thanks!

**Comment by cwanja** [comment, marked as answer]
As long as your Tesla and TeslaMate continues to be connected to the internet, they do not need to be located near one another. 

Individuals are running TeslaMate in data centers not near their cars and have zero issues.

**Comment by rfgmacros** [comment]
Thanks! I wasn't sure if it only collected data when on the local wifi. This will put me more at ease with the car so far away.

**Reply by cwanja** [reply]
For what it's worth, TeslaMate does not talk to your Tesla vehicle directly.

**Reply by rfgmacros** [reply]
I think I knew that at some level, but not being overly technical (shell scripting is about where I tap out :)), I wasn't 100% sure how the communication occurred.
